File: Examen I bimestre/Final/app/src/main/java/com/example/afinal/Pokemon.kt
package com.example.afinal

class Pokemons(var id: Int,var nombre:String,
               var tipo:String, var nivel: String,
               var activo: String, var entrenador: String,
var latitud: String, var longitud: String,var url:String, var imagen: String) {
    override fun toString(): String {
        return "id: ${id}, nombre:${nombre}, tipo: ${tipo}, nivel:${nivel}, entrenador:${entrenador}"
    }


}

File: Examen I bimestre/Backend/pokemon/api/models/Pokemon.js
/**
 * Pokemon.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {


  attributes: {

    nombre:{
      type: 'string',
    },
    tipo:{
      type: 'string',
    },
    nivel:{
      type: 'string',

    },

    entrenador:{
      type: 'string',

    },
    latitud :{
      type: 'string',

    },
    longitud:{
      type: 'string',

    },
    url:{
      type: 'string',

    },
    imagen:{
      type: 'string',

    },


/*
    entrenador: { // Many to One (nombre FK) - mismo nombre q la relacion
      model: 'entrenador',
     // required: true // (Es opcional 1 muchos 0 muchos)
    }

*/
  },

};

